lib/otel/segments/http-external.js

```diff
@@ -9,12 +9,13 @@ const NAMES = require('../../metrics/names')
 const recordExternal = require('../../metrics/recorders/http_external')
 
 const {
-  ATTR_HTTP_HOST
+  ATTR_NET_PEER_NAME,
+  ATTR_SERVER_ADDRESS
 } = require('../constants')
 
 module.exports = function createHttpExternalSegment(agent, otelSpan) {
   const context = agent.tracer.getContext()
-  const host = otelSpan.attributes[ATTR_HTTP_HOST] || 'Unknown'
+  const host = otelSpan.attributes[ATTR_SERVER_ADDRESS] || otelSpan.attributes[ATTR_NET_PEER_NAME] || 'Unknown'
   const name = NAMES.EXTERNAL.PREFIX + host
```

Comment:
so it also appears this is incorrect and a complete oversight.  the name should be what's there + ` parsed.path = urltils.obfuscatePath(config, parsed.path)`